pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1687,6 +1687,11 @@ func (og *operationGenerator) nodeExpandVolume(
 				// On retry, NodeExpandVolume will be called again but do nothing
 				return false, fmt.Errorf("mountVolume.NodeExpandVolume update PVC status failed : %v", err)
 			}
+			// Get the updated pvc and add resize complete annotation to it
+			pvc, err = og.kubeClient.CoreV1().PersistentVolumeClaims(pv.Spec.ClaimRef.Namespace).Get(context.TODO(), pv.Spec.ClaimRef.Name, metav1.GetOptions{})
+			if err != nil {
+				return false, fmt.Errorf("mountVolume.NodeExpandVolume get PVC failed : %v", err)
+			}
```

Comment:
Why are we doing this?

Reply:
These are from the previous design that used PVC annotation to communicate with the external resizer. These are no longer necessary. I will remove them.